bin/smart-proxy-openscap-send

```diff
@@ -37,8 +37,8 @@ end
 
 begin
   Proxy::OpenSCAP::send_spool_to_foreman
-rescue StandardError => e
-  logger.debug e.backtrace.join("\n\t")
-  logger.error "Failed to send SCAP results to the Foreman server: #{e}"
+rescue StandardError, RuntimeError => e
```

Comment:
`StandardError` will catch `RuntimeError` as well (it inherits from it) so it doesn't have to be mentioned here